cit-api/pipeline/urls.py

```diff
@@ -45,5 +45,5 @@
     url(r"^civicleaders/$", pipeline.views.general.CivicLeaderList.as_view()),
     url(r"^pageviews/$", pipeline.views.general.PageViewList.as_view()),
     
-    url(r"^investments/$", pipeline.views.investment.InvestmentsList.as_view(), name="investments")
+    url(r"^opportunities/$", pipeline.views.opportunity.OpportunitiesList.as_view(), name="opportunities")
```

Comment:
should this be a separate app? I don't necessarily believe this belongs in the pipeline app.

Reply:
It doesn't really make sense belonging to the pipeline app, but having two apps using the same database in Django seems like it would be challenging to manage, and not necessarily the way we want to go: https://www.reddit.com/r/django/comments/akp761/migration_problems_with_two_projects_that_have/ef6t0zi/.

I'm going to leave this as is for now.